packages/languages/bin/download.js

```diff
@@ -1,24 +1,21 @@
+/* eslint-disable */
+
 /**
  * External dependencies
  */
 const https = require( 'https' );
 const fs = require( 'fs' );
+const path = require( 'path' );
 
 const LANGUAGES_META_URL = 'https://widgets.wp.com/languages/calypso/languages-meta.json';
-const FILE_PATH = 'client/languages/languages-meta.json';
+const FILE_PATH = path.resolve( __dirname, '..', 'src', 'languages-meta.json' );
```

Comment:
Nit: `path.resolve(__dirname, "../src/languages-meta.json")` does the same (both in unix and win32) and I find it a bit less confusing